Gemfile.lock

```diff
@@ -177,9 +177,7 @@ GEM
     zeitwerk (2.6.12)
 
 PLATFORMS
-  arm64-darwin-22
-  arm64-darwin-23
-  x86_64-linux
```

Comment:
We should keep all of these in addition to the newly added `x86_64-darwin-20`

Reply:
The rest looks spot on! Nice work

Reply:
Re-added!